src/services/SupabaseDataService.ts

```diff
@@ -247,50 +248,89 @@ export class SupabaseDataService extends BaseSupabaseService<KyselyDataDatabase>
     tokenIds: string[];
     chainId: number;
   }) {
-    console.log("[marketplace-api] Validating orders by token IDs", tokenIds);
+    console.log(
+      "[marketplace-api] Validating orders by token IDs",
```

Comment:
We follow a different pattern for logs, which you do apply in the Error thrown below

Reply:
Ended up removing this log as it's noisy and superfluous.